### DISCUSSION TITLE
2.03版本无法显示qrcord

### DISCUSSION THREAD

**Discussion by zjc0424** [messense/aliyundrive-webdav#746] [Q&A]
![image](https://user-images.githubusercontent.com/29698700/223330784-f00dc0d1-4f11-4ce0-841c-195a61293975.png)
2.03版本无法显示qrcord

**Comment by messense** [comment]
可以试试用在线工具获取：https://github.com/messense/aliyundrive-webdav#%E8%8E%B7%E5%8F%96-refresh-token
